modules/installation-about-custom-gcp-vpc.adoc

```diff
@@ -0,0 +1,52 @@
+
+[id="installation-about-custom-gcp-vpcs-requirements_{context}"]
+== Requirements for using your VPC
+
+The installation program will no longer create the following components:
+* VPC
+* Subnets
+* Cloud Router
+* Cloud NAT
+* NAT IP addresses
+
+If you use a custom VPC, you must correctly configure it and its subnets for the installation program and the cluster to use. The installation program cannot subdivide network ranges for the cluster to use, set route tables for the subnets, or set VPC options like DHCP, so you must do so before you install the cluster.
+
+Your VPC and subnets must meet the following characteristics:
+
+* The VPC must be in the same GCP project that you deploy the {product-title} cluster to.
+* To allow access to the internet from the control plane and compute machines, you must configure Cloud NAT on the subnets to allow egress to it. These machines do not have a public address. Even if you do not require access to the internet, you must allow egress to the VPC network to obtain the installation program and images. Because multiple Cloud NATs cannot be configured on the shared subnets, the installation program cannot configure it.
+
+To ensure that the subnets that you provide to the installation program are suitable, it confirms the following data:
+
+* All the subnets that you specify exist and belong to the VPC that you specified.
+* The subnet CIDRs belong to the machine CIDR.
+* You must provide a subnet to deploy the cluster control plane and compute machines to. You can use the same subnet for both machine types.
+
+If you destroy a cluster that uses an existing VPC, the VPC is not deleted.
+
+[id="installation-about-custom-gcp-permissions_{context}"]
+== Division of permissions
+
+Starting with {product-title} 4.3, you do not need all of the permissions that are required for an installation program-provisioned infrastructure cluster to deploy a cluster. This change mimics the division of permissions that you might have at your company: some individuals can create different resource in your clouds than others. For example, you might be able to create application-specific items, like instances, buckets, and load balancers, but not networking-related components such as VPCs, subnet, or ingress rules.
+
+The GCP credentials that you use when you create your cluster do not need the networking permissions that are required to make VPCs and networking core components within the VPC, such as subnets, routing tables, internet gateways, NAT, and VPN. You still need permission to make the application resources that the machines within the cluster require, such as load balancers, security groups, storage, and nodes.
```

Comment:
Can you break this sentence up or shorten it? It didn't register with me at first that you are making core components from VPCs and networking. I read "networking core components" as `adjective-adjective-noun`, rather than `noun adjective-noun`.
Also, this phrase in the second sentence is confusing: "to make the application resources that the machines within the cluster require...." I'm not exactly sure what you are trying to say here.